models/errors.go

```diff
@@ -0,0 +1,79 @@
+package models
+
+import (
+	"context"
+	"errors"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+)
+
+type ErrorList struct {
```

Comment:
I think something like `ErrorResponse` or `ErrorEntity` or `ErrorBody` would be a better name as its not strictly a list - it contains a list. But that is me being very nit picky.